hack/verify-generators.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT="$(dirname "${BASH_SOURCE}")/.."
```

Comment:
This is the parent dir of the script root. Did you mean to put `..` on the line below?

Reply:
Yes, I just copied this from existing scripts.

Removed this in 13d396b2b8fa068a2cea08e76f30b02f9fa39308